src/command.js

```diff
@@ -221,4 +228,31 @@ export default class StorybookCommand extends Command {
       await page?.close();
     }
   }
+
+  async getStorybookVersion(url) {
+    let version, page;
+
+    try {
+      page = await this.percy.browser.page();
+      this.log.debug(`Get Storybook version: ${url}/?path=/settings/about`);
+
+      await page.goto(`${url}/?path=/settings/about`);
```

Comment:
We should construct this URL like we do other URLs, like so:

```js
let aboutUrl = new URL('?path=/settings/about', url);
```

It serves to both validate the provided URL, and ensure we aren't accidentally creating an invalid or incorrect URL (such as double `//` in the middle, or if there are existing URL fragments)